app/scripts/modules/core/src/application/service/applicationDataSource.ts

```diff
@@ -64,6 +64,11 @@ export interface IDataSourceConfig<T> {
    */
   icon?: string;
 
+  /**
+   * Represents the name of the svg to be used with the svg loader (Icon.tsx)
+   */
+  iconSvg?: string;
```

Comment:
Instead of `string` here and in `navigationCategory`, let's use the `IconNames` type from `core/presentation` so we can strongly type this field going forward

Reply:
Great call out, I did not we had this type but I am a huge fan of strongly typed enums!